urllib3/_collections.py

```diff
@@ -167,7 +167,7 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self.__eq__(other)
 
-    if not PY3: # Python 2
+    if not PY3:  # Python 2
```

Comment:
That can be avoided by adding appveyor builds covering the win32 specific code paths, and merging all job coverage on codecov.io which can notify the total coverage across all jobs.